_j/mkl_threads.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Consider adjusting ``OMP_NUM_THREADS`` environment variable for containerized deployments
+---
+
+Consider adjusting ``OMP_NUM_THREADS`` environment variable for containerized
+deployments, one or more libraries use Intel's MKL that does not detect
+correctly one or more libraries use Intel's MKL that does not detect correctly
+resource allocation in the cluster resource allocation in the cluster.
+
+## Issue description
+
+OpenMKL detects number of CPUs present on a machine where the application runs.
+This generally does not need to correspond to number of CPUs allocated for
+containerized applications. Refer to [Kubernetes documentation][4] to get more
+information about assigning CPU requests to a pod.
+
+## Affected packages:
+
+ * ``intel-tensorflow``
+ * ``pytorch``
+
+## Severity
+
+ * WARNING
+
+## Issue fix
+
+Make sure the ``OMP_NUM_THREADS`` environment variable is set and corresponds
+to number of CPUs allocated for the container.
+
+## Pipeline units
+
+ * MKLThreadsWrap
+
+## Recommendation types
+
+ * latest
+ * performance
+ * secure
+ * stable
+ * testing
```

Comment:
all reccomendations? Don't we have docs to be linked for each typo of recommendation?

Reply:
:)